blackbelt/handle_github.py

```diff
@@ -270,6 +270,30 @@ def deploy(pr_url):
     try:
         ticket_id = get_pr_ticket_id(merge_info['description'])
         move_to_deployed(card_id=ticket_id, comment=comment)
+        create_release(ref=merge_info['sha'], payload='', description="Deployed to production")
```

Comment:
Agreed, it should have its own try (and doesn't make sense to open in browser if it fais.